### PR TITLE
Adding Transfers End-to-end architecture

### PR DESCRIPTION
**Pull request by elnyry**
Summary of changes:
-  Adding Transfers End-to-end architecture diagram
- Corresponding updates to the Readme doc

### CODE REVIEW

File: mojaloop-technical-overview/central-ledger/README.md [hunk 1]
```diff
@@ -10,11 +10,17 @@ The central ledger is a series of services that facilitate clearing and settleme
 
 ### 1.1 Architecture overview
 
+* [Central-Ledger Architecture](assets/diagrams/architecture/Arch-mojaloop-central-ledger.svg)
```
Comment (mdebarros) [inline mojaloop-technical-overview/central-ledger/README.md]: Why the link?

File: mojaloop-technical-overview/central-ledger/README.md [hunk 2]
```diff
@@ -10,11 +10,17 @@ The central ledger is a series of services that facilitate clearing and settleme
...
+
 ![Central-Ledger Architecture](assets/diagrams/architecture/Arch-mojaloop-central-ledger.svg)
 
-* [Central-Ledger Architecture](assets/diagrams/architecture/Arch-mojaloop-central-ledger.svg)
+## 2. Transfers End-to-End Architecture
+
+* [Transfers Architecture ](assets/diagrams/architecture/Transfers-Arch-End-to-End.svg)
```
Comment (mdebarros) [inline mojaloop-technical-overview/central-ledger/README.md]: Why the link?

File: mojaloop-technical-overview/central-ledger/README.md (hunk 1 above)
Comment (elnyry) [reply mojaloop-technical-overview/central-ledger/README.md]: It was already there. Moreover, why not?

File: mojaloop-technical-overview/central-ledger/README.md (hunk 2 above)
Comment (elnyry) [reply mojaloop-technical-overview/central-ledger/README.md]: Following the same pattern as the other two diagrams..

File: mojaloop-technical-overview/central-ledger/README.md (hunk 1 above)
Comment (elnyry) [reply mojaloop-technical-overview/central-ledger/README.md]: It was already there at the bottom, I moved it to appear in front of the picture.. may be the link is not needed

File: mojaloop-technical-overview/central-ledger/README.md (hunk 2 above)
Comment (elnyry) [reply mojaloop-technical-overview/central-ledger/README.md]: Following the same pattern as the existing ones.. can change it if link is not needed

**Pull request merged**